tests/foreman/api/test_computeresource_gce.py

```diff
@@ -227,7 +227,7 @@ def google_host(self, googleclient):
     @pytest.mark.e2e
     @pytest.mark.tier1
     @pytest.mark.pit_server
-    @pytest.mark.build_sanity
+    # @pytest.mark.build_sanity
```

Comment:
@shweta83 Can't we skip only for the ipv6?

Reply:
@devendra104 @shweta83 There is the other way we can use is:
1. File a Jira for the issue 
2. Use the `is_open` function within test function to skip the test execution as below:
```
@pytest ......
def test_positive_gce_host_provisione(.......
"""Host ........
if is_open('SAT-<issue_id>') and settings.server.is_ipv6:
    pytest.skip('Skipping for Ipv6 due to SAT-<issue_id>')
```

Reply:
Can't we use skipif marker?

Reply:
@shweta83 With only ipv6 setting, this would be skipped unless we explicitly remove it which we tend to forget in general.

Combine it with `Is_open` jira with `and` operator.

Reply:
@jyejare I intentionally skipped the above implementation as it would unnecessarily execute all the fixtures and setup just to skip the whole test. Also the test fails in setup where we are calling Google APIs so it would be better to skip this way.

Reply:
@shweta83 Nop, I am talking using `skipif` marker itself to check both the things.
```
@pytest.mark.skipif((settings.server.is_ipv6 and is_open('SAT-1234')), reason='Google CR APIs failing for IPv6')
```

Reply:
Updated.